Makefile

```diff
@@ -228,17 +228,13 @@ $(GOPATH1)/bin/%:
 	cp -f $< $@
 
 test: build
-	go test $(GOTAGS) -race $(UNIT_TEST_SOURCES) -timeout 3600s | logfilter
+	go test $(GOTAGS) -race $(UNIT_TEST_SOURCES) -timeout 1h -coverprofile=coverage.txt -covermode=atomic
```

Comment:
could you store the output `coverage.txt` in a place *other* then the repo's root ? maybe ~/coverage.txt ?
( otherwise, we would also need to add this to the `.gitignore` )

Reply:
I think gitignore is better than putting stuff in people's home directory. I also added some of the other files missing from the cover/Prof targets